docs/devguide/migrate-dashboards.asciidoc

```diff
@@ -0,0 +1,43 @@
+== Migrating dashboards from Kibana 5.x to 6.x
+
+Starting with Kibana 6.0, the format of the dashboards changes and it's not compatible with the Kibana 5.x dashboards.
+
+Each module keeps the Kibana 5.x dashboards under a 5.x directory, and under the default directory for Kibana 6.0 or greater. 
+
+The 5.x directory should contain the following directories:
+- search
+- visualization
+- dashboard
+- index-pattern
+
+The default directory should contain the following directories:
+- dashboard
+- index-pattern
+
+To collect all the dashboards per module to the global `_meta/kibana` director, you would need to run `make collect`.
+The index-pattern directory is generated from fields.yml by the `make update` command. 
+
+To migrate your Kibana 5.x dashboards to Kibana 6.0 and above, you can import the dashboards into Kibana 5.6 and then
+exporting them using Beats 6.0 version.
+
+1. Start Kibana 5.6
+2. Import Kibana 5.x dashboards using Beats 6.0 version. Before importing the dashboards, make sure you run `make
+update` in the Beat directory.
+
+[source,shell]
+-----------------
+$ make update
+$ ./metricbeat setup -E setup.dashboards.directory=_meta/kibana
+-----------------
+3. Export the dashboards using Beats 6.0 version.
+
+[source,shell]
+-----------------
+$ cd beats/dev-tools/cmd/dashboards
+$ ./export_dashboards -dashboards Metricbeat-Rabbitmq
+-----------------
+
+This will save a single JSON file that contains the dashboard and all the dependencies like searches, visualizations.
```

Comment:
Change to: The `export_dashboards` tool creates....." or "This command creates...." [not sure if it's best to call this a tool or what...just need a noun because "this" has an unclear antecedent]